### ISSUE TITLE
Add docs from related initiatives in domain such as from EOSC 

### ISSUE THREAD

**Issue by roelandordelman** [CLARIAH/clariah-plus#110]
e.g., [Quality Assurance Baseline Criteria for Research Projects](http://hdl.handle.net/10261/160086)

**Comment by proycon** [comment]
I assume your aim here is just to link to interesting related initiatives so we can all be aware of them? We can use this issue to track that.

The one your mention is an interesting document indeed, I believe it was mentioned by @4tikhonov at the Tech Day. I certainly agree with it contents. There's a certain degree of overlap between this document and what we're doing in the software requirements, although there's also a bit of a difference in focus and specificity/verbosity.

Other notable documents:
* [Guide to projects and software development at the Netherlands eScience Center](https://guide.esciencecenter.nl/#/README)
* [Software Quality Guidelines](https://github.com/CLARIAH/software-quality-guidelines/) - Our earlier work in CLARIAH-CORE (more verbose)
* [Software Evaluation: Criteria-based Assessment](http://software.ac.uk/sites/default/files/SSI-SoftwareEvaluationCriteria.pdf) - Software Sustainability Institute
* [ISO/IEC 25010:2011
Systems and software engineering — Systems and software Quality Requirements and Evaluation (SQuaRE) — System and software quality models](https://www.iso.org/standard/35733.html) - Behind a stupid paywall as ISO usually does :(

**Comment by roelandordelman** [comment]
@proycon yes, for us to check if we missed something or notice a possible misalignment, for readers to show that our requirements are aligned with initiatives in the field.

**Comment by menzowindhouwer** [comment]
[Fair metrics for FAIR software](https://www.nwo.nl/en/open-science-practice-webinar-series#Fair%20metrics%20for%20FAIR%20software)

**Comment by proycon** [comment]
Perhaps we might want to cite a few of these as related work in the software/infrastructure/data requirements eventually

**Comment by proycon** [comment]
I also want to add this to the list: https://fair-software.eu/
They posit five [recommendations](https://fair-software.eu/recommendations/repository) for fair software:

1. Use a publicly accessible repository REPOSITORY with version control
2. Add a license
3. Register your code in a community registry
   * Here they link to https://github.com/NLeSC/awesome-research-software-registries
4. Enable citation of the software
  * Here they refer to codemeta (which is what we use) and citation.cff 
  * They also mention archiving, where they refer to for instance zenodo
5. Use a software quality checklist
  * Here they refer to various checklists (including the one we developed in CLARIAH-CORE)
